### ISSUE TITLE
html5lib requires setuptools version 18.5 or above error 

### ISSUE THREAD

**Issue by yagi815** [ckan/ckan#3232]
Hi all 
I trying to install ckan 2.5.2 on rhel 6.6.
I have "html5lib requires setuptools version 18.5 or above" error message When I execute  (default)# pip install --ignore-installed -r default/src/ckan/pip-requirements-docs.txt .
Is there any solution for ti?

I refer this installation guide : https://github.com/ckan/ckan/wiki/How-to-install-CKAN-2.5.2-on-CentOS-6.8

**Comment by torfsen** [comment]
This should fix your problem:
```
pip install --upgrade pip setuptools
```

**Comment by amercader** [comment]
We updated the bleach requirement to solve a related issue (69408d2). Please use the `release-v2.5.3` branch and see if that solves the issue

**Comment by bartmika** [comment]
+1

**Comment by rodolfomartinez** [comment]
Big +1

**Comment by amercader** [comment]
@bartmika @rodolfomartinez +1 as in "I have the same issue" or as in "Your suggestion fixed it"?

**Comment by bartmika** [comment]
@amercader It appears the code written by @torfsen has helped:

```
pip install --upgrade pip setuptools
```

**Comment by amercader** [comment]
Fixed in 2.5.3
